integration/conftest.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2017 Google Inc.
+# Copyright 2022 Google Inc.
```

Comment:
We only set this for new files (this is the creation date)

Reply:
Got it, thought this was last time edited. I'll keep that in mind!